integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -207,13 +232,23 @@ spec:
 {{- include "alluxio.worker.secretVolumes" . }}
           {{- end }}
         {{- end }}
-        {{- if .Values.tieredstore }} 
+        {{- if .Values.tieredstore }}
 {{- include "alluxio.worker.tieredstoreVolumes" . }}
         {{- end}}
-        {{- if .Values.mounts }}   
+        {{- if .Values.mounts }}
           {{- range .Values.mounts }}
         - name: "{{ .name }}"
           persistentVolumeClaim:
             claimName: "{{ .name }}"
           {{- end }}
         {{- end }}
+        {{- if .Values.worker.fuseEnabled }}
+        - name: alluxio-fuse-device
+          hostPath:
+            path: /dev/fuse
+            type: CharDevice
+        - name: alluxio-fuse-mount
+          hostPath:
+            path: {{ .Values.fuse.mountPath | dir }}
+            type: DirectoryOrCreate
```

Comment:
Note that `hostPath` that are created by `DirectoryOrCreate` are only writable by `root` user
Ref: https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

Reply:
@LuQQiu

Reply:
Emmm... by using fuse, the pod needs to run as root
```
      securityContext:
        runAsUser: 0
        runAsGroup: 0
        fsGroup: 0
```
so that it's fine that `DirectoryOrCreate` can only writable by root user.
And interestingly, the mount point is actually accessible by other users in my testing. 
I can double-check this part

Reply:
We can change to Directory here. in our testing, we create the mount folder first